update README format and repo requirements (#93)
redhat-cop/cloud.vmware_ops: README.md

# Ansible Collection: cloud.vmware_ops

> This repo hosts the `cloud.vmware_ops` validated Ansible Collection.

Reviewer: repo ->repository

## Use Cases

* Use Case Name: Create and Modify Cluster Settings and Perform ESXi Maintenance
  * Actors:
    * System Admin
  * Description:
    * A systems administrator can modify various vCenter cluster settings.
  * Related Playbooks:
    * cluster_settings/disable_high_availability.yml
    * cluster_settings/enable_high_availability.yml
    * cluster_settings/manage_all_settings.yml
    * esxi_management/disable_maintenance_mode.yml
    * esxi_management/enable_maintenance_mode.yml
  * Roles:
    * `cloud.vmware_ops.cluster_settings` - Set DPM, DRS, vCLS, HA, and vSAN settings
    * `cloud.vmware_ops.esxi_maintenance_mode` - Manage the maintenance status of an ESXi host
  * Flow:
    - Using the `manage_all_settings` playbook, control a variety of settings in vCenter
    - Quickly enable or disable high availability on the cluster using the dedicated playbooks
    - Put an ESXi host in maintenance mode to adjust settings on the host, then add it back into production using the `esxi_maintenance_mode` role
    - When a new ESXi host or datastore has been added to the cluster, include the `cluster_settings` role to adjust related settings.

> * Use Case Name: Create and Manage ESXi Hosts in A Nested VCenter

Reviewer 2: ```suggestion
* Use Case Name: Create and Manage ESXi Hosts and Nested VCenter
```

  * Actors:
    * System Admin
  * Description:
    * A systems administrator can deploy a virtual vSphere environment for testing or isolation purposes.
  * Related Playbooks:
    * esxi_management/add_esxi_host_to_vcenter.yml
    * esxi_management/remove_esxi_host_from_vcenter.yml
    * esxi_management/reconnect_esxi_host_in_vcenter.yml
    * provision_vcenter/provision_vcsa_on_esxi.yml
    * system_settings.yml
  * Roles:
    * `cloud.vmware_ops.provision_virtual_esxi` - Provision a set of virtual ESXi hosts on an existing vSphere environment
    * `cloud.vmware_ops.vcenter_host_connection` - Manage the connection status of an ESXi host to a vCenter appliance

>     * `cloud.vmware_ops.system_settings` - Manage the system settings of a vCenter appliance

Reviewer 2: `provision_vcenter/provision_vcsa_on_esxi.yml `playbook uses `cloud.vmware_ops.provision_vcenter` role

    - Deploy a vCenter appliance to one of the new virtual hosts using the `provision_vcenter` role or `provision_vcsa_on_esxi` playbook
    - Add the other virtual ESXi hosts to the new vCenter using the `add_esxi_host_from_vcenter` playbook or `vcenter_host_connection` role
    - Modify vCenter appliance system settings using the `system_settings` playbook or `system_settings` role

* Use Case Name: Manage a VM and Export and/or Deploy as an OVF
  * Actors:
    * System Admin
  * Description:
    * A systems administrator can create or modify an existing VM, export it as an OVF, and deploy the OVF to other clusters.
  * Related Playbooks:
    * provision_vm/manage_vm.yml
    * provision_vm/deploy_ovf.yml
    * export_vm_as_ovf.yml
    * snapshot_management/create_snapshot.yml
    * snapshot_management/remove_all_snapshots.yml
    * snapshot_management/remove_snapshot.yml
    * snapshot_management/revert_to_a_snapshot.yml
  * Roles:
    * `cloud.vmware_ops.export_vm_as_ovf` - Exports an existing VM as an OVF file
    * `cloud.vmware_ops.provision_vm` - Create, manage, or delete a VM
    * `cloud.vmware_ops.snapshot_management` - Manage the snapshots of a VM

>     * `cloud.vmware_ops.manage_folder` - Create or delete a folder

Reviewer 2: `provision_vm/deploy_ovf.yml` playbook uses `deploy_ovf` role

  * Flow:
    - Create or adjust a VM to prepare it for export using the `provision_vm` role or `manage_vm` playbook
    - Optionally take snapshots of the VM using the `snapshot_management` role or `create_snapshot` playbook

>     - Power off the VM using `provision_vm` role or `manage_vm` playbook

Reviewer 2: Are you sure that `manage_vm` playbook can power off VM, since it misses `provision_vm_state` var

Author: Yes, the user can specify that variable at runtime

    - Export the VM as an OVF file using the `export_vm_as_ovf` playbook or `export_vm_as_ovf` role

>     - Deploy the VM to a new cluster using the `deploy_ovf` playbook or `provision_vm` role

Reviewer 2: ```suggestion
    - Deploy the VM from an OVF file using the `deploy_ovf` playbook or `deploy_ovf` role
```
